exchanges/gateio/gateio_wrapper.go

```diff
@@ -2573,3 +2589,49 @@ func getSettlementCurrency(p currency.Pair, a asset.Item) (currency.Code, error)
 	}
 	return currency.EMPTYCODE, fmt.Errorf("%w: %s", asset.ErrNotSupported, a)
 }
+
+// WebsocketSubmitOrders submits multiple orders to the exchange through the websocket
+func (g *Gateio) WebsocketSubmitOrders(ctx context.Context, orders []*order.Submit) ([]*order.SubmitResponse, error) {
+	var a asset.Item
+	for x := range orders {
+		if err := orders[x].Validate(g.GetTradingRequirements()); err != nil {
+			return nil, err
+		}
+
+		if a == asset.Empty {
+			a = orders[x].AssetType
+			continue
+		}
+
+		if a != orders[x].AssetType {
+			return nil, fmt.Errorf("%w %v", errSingleAssetRequired, a)
+		}
+	}
+
+	if !g.CurrencyPairs.IsAssetSupported(a) {
+		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, a)
+	}
+
+	switch a {
+	case asset.Spot:
+		reqs := make([]*CreateOrderRequest, len(orders))
+		for x := range orders {
+			var err error
+			reqs[x], err = g.getSpotOrderRequest(orders[x])
+			if err != nil {
+				return nil, err
+			}
+		}
+		got, err := g.WebsocketSpotSubmitOrders(ctx, reqs...)
```

Comment:
🦍 This is the resp, from the perspective of this func
```suggestion
		resp, err := g.WebsocketSpotSubmitOrders(ctx, reqs...)
```